test/TransitivePathTest.cpp

```diff
@@ -344,94 +330,99 @@ TEST_P(TransitivePathTest, rightBoundToVar) {
+                                         {V(2), V(4)},
+                                         {V(2), V(7)},
+                                         {V(4), V(7)},
+                                         {V(4), V(0)},
+                                         {V(3), V(3)},
+                                         {V(7), V(0)},
+                                         {V(7), V(2)},
+                                         {V(7), V(7)},
+                                         {V(10), V(11)}});
 
   TransitivePathSide left(std::nullopt, 0, Variable{"?start"}, 0);
   TransitivePathSide right(std::nullopt, 1, Variable{"?target"}, 1);
   auto T =
       makePathUnbound(std::move(sub), {Variable{"?start"}, Variable{"?target"}},
                       left, right, 1, 2);
   auto resultTable = T->computeResultOnlyForTesting();
-  assertSameUnorderedContent(expected, resultTable.idTable());
+  ASSERT_THAT(resultTable.idTable(),
+              ::testing::UnorderedElementsAreArray(expected));
 }
 
 TEST_P(TransitivePathTest, maxLength2FromId) {
-  IdTable sub(2, makeAllocator());
-  sub.push_back({V(0), V(2)});
-  sub.push_back({V(2), V(4)});
-  sub.push_back({V(4), V(7)});
-  sub.push_back({V(0), V(7)});
-  sub.push_back({V(3), V(3)});
-  sub.push_back({V(7), V(0)});
-  // Disconnected component.
-  sub.push_back({V(10), V(11)});
-
-  IdTable expected(2, makeAllocator());
-
-  expected.push_back({V(7), V(0)});
-  expected.push_back({V(7), V(2)});
-  expected.push_back({V(7), V(7)});
+  auto sub = makeIdTableFromVector({
+      {V(0), V(2)},
+      {V(2), V(4)},
+      {V(4), V(7)},
+      {V(0), V(7)},
+      {V(3), V(3)},
+      {V(7), V(0)},
+      // Disconnected component.
+      {V(10), V(11)},
+  });
+
+  auto expected = makeIdTableFromVector({
+      {V(7), V(0)},
+      {V(7), V(2)},
+      {V(7), V(7)},
+  });
 
   TransitivePathSide left(std::nullopt, 0, V(7), 0);
   TransitivePathSide right(std::nullopt, 1, Variable{"?target"}, 1);
   auto T =
       makePathUnbound(std::move(sub), {Variable{"?start"}, Variable{"?target"}},
                       left, right, 1, 2);
   auto resultTable = T->computeResultOnlyForTesting();
-  assertSameUnorderedContent(expected, resultTable.idTable());
+  ASSERT_THAT(resultTable.idTable(),
+              ::testing::UnorderedElementsAreArray(expected));
 }
 
 TEST_P(TransitivePathTest, maxLength2ToId) {
-  IdTable sub(2, makeAllocator());
-  sub.push_back({V(0), V(2)});
-  sub.push_back({V(2), V(4)});
-  sub.push_back({V(4), V(7)});
-  sub.push_back({V(0), V(7)});
-  sub.push_back({V(3), V(3)});
-  sub.push_back({V(7), V(0)});
-  // Disconnected component.
-  sub.push_back({V(10), V(11)});
-
-  IdTable expected(2, makeAllocator());
-
-  expected.push_back({V(0), V(2)});
-  expected.push_back({V(7), V(2)});
+  auto sub = makeIdTableFromVector({
+      {V(0), V(2)},
+      {V(2), V(4)},
+      {V(4), V(7)},
+      {V(0), V(7)},
+      {V(3), V(3)},
+      {V(7), V(0)},
```

Comment:
This change (replace V(x) by x) can probably be done by a rather simple regex rename:)